lib/LanguageServerRename/Model/Renamer.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerRename\Model;
+
+use Generator;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\ByteOffsetRange;
+use Phpactor\TextDocument\TextDocument;
+
+interface Renamer
+{
+    public function getRenameRange(TextDocument $textDocument, ByteOffset $offset): ?ByteOffsetRange;
+    /** @return Generator<RenameResult> */
+    public function rename(TextDocument $textDocument, ByteOffset $offset, string $newName): Generator;
```

Comment:
not important (and really CS fixer should fix it) but docblocks should be multiline:

```
/**
 * @return ...
 */
```

and there should be exactly one space between member definitions

Reply:
You know, I've been missing help with this. I've used extensions which when inside `/**` and you hit enter, you get in the proper indent with ` * ` already typed, getting to:
```php
    /**
     * <>
```